src/commcare_cloud/fab/operations/offline.py

```diff
@@ -35,7 +36,13 @@ def prepare_files():
     })))
 
     # NPM
-    local('cd {}/commcare-hq && npm install --production'.format(OFFLINE_STAGING_DIR))
+    local('cd {}/commcare-hq'.format(OFFLINE_STAGING_DIR))
+
+    yarn_lock = os.path.join(env.code_root, YARN_LOCK)
+    if files.exists(yarn_lock):
```

Comment:
I might have missed this in another PR description, but is the plan to replace both bower and npm with yarn?

Reply:
yes, `yarn` runs on top of `npm` and effectively works with both `npm` and `bower` repositories

Reply:
So we're modernizing *and* using one fewer tool? Fantastic.

Reply:
yes, except `yarn 2` and `npm 2` are on the way...but baby steps...